src/main/java/com/hotel/management/hotel/quarto/domain/QuartoForm.java

```diff
@@ -56,5 +52,33 @@ public QuartoForm estado(Estado estado) {
     public void apply() {
         this.action.accept(this);
     }
-    
+
+    public QuartoForm data(Date data) {
+        this.data = data;
+        return this;
+    }
+
+    public Double getPrecoPorNoite() {
```

Comment:
1 - Se é get não pode retornar null, por que temos que evitar o NPE [Null Object](https://justgokus.medium.com/what-is-the-null-object-design-pattern-d8bf2c5333b6)
2 - Get e set tem que ser gerado pelo lombok. @Getter